src/renderers/dom/client/wrappers/ReactDOMInput.js

```diff
@@ -67,6 +70,14 @@ var ReactDOMInput = {
         props,
         inst._currentElement._owner
       );
+
+      if (props.valueLink !== undefined && !didWarnValueLink) {
+        warning(
+          false,
+          '`.valueLink` on `input` is deprecated; set `.value` and `.onChange` instead.'
```

Comment:
Removed dots, added checked link.

Didn't want to put the warnings in LinkedValueUtils because I'm expecting that that class will be used in the addons/wrapper for anyone who wants to reuse the valuelink behavior in their apps/components, and they won't want to see the warning.